src/components/Permalink/Permalink.js

```diff
@@ -44,6 +45,33 @@ const defaultProps = {
   departuresFilter: undefined,
 };
 
+const redirectToDraw = drawId => {
+  const urlParams = qs.parse(window.location.search);
+
+  if (urlParams.z) {
+    // Convert the zoom level to match the different scale on the old wkp.
+    urlParams.zoom = layerHelper.convertToOldZoom(parseInt(urlParams.z, 10));
+    delete urlParams.z;
+  }
+
+  if (urlParams.x || urlParams.y) {
+    // Reproject the coordinates to the old wkp projection: EPSG:21781.
+    const [newX, newY] = transform(
+      [parseInt(urlParams.x, 10), parseInt(urlParams.y, 10)],
+      'EPSG:3857',
+      'EPSG:21781',
+    );
+    urlParams.x = newX;
+    urlParams.y = newY;
+  }
+
+  urlParams['wkp.draw'] = drawId;
+
+  window.location.href = `http://wkp.prod.trafimage.geops.ch/?debug#/ch.sbb.netzkarte.draw?${qs.stringify(
```

Comment:
No `?debug` and please make this configurable.

Reply:
I use configurable 'appBaseUrl'  now